scripts/docker/README.md

```diff
@@ -0,0 +1,36 @@
+<!---
+  Copyright 2022 SECO Mind Srl
+
+  SPDX-License-Identifier: Apache-2.0
+-->
+
+# Astarte Message Hub Container
+
+This README explains how to build and run the Astarte Message Hub container. Follow the steps below
+to set up your environment and get the container running with the appropriate configuration.
+
+## Prerequisites
+
+Ensure you have Docker installed on your machine.
+
+## Build the Container
+
+1. Clone the repository containing the Astarte Message Hub code.
+2. Navigate to the root directory of the repository.
+3. Run the build script to create the Docker image:
+   ```sh
+   ./scripts/docker/build.sh
+   ```
+
+## Run the Container
+
+To run the container with your configuration file:
+
+1. Ensure you have your configuration file named `config.toml`.
+2. Run the Docker container, mounting the configuration file and opening the gRPC port:
+
+```sh
+docker run -p 50051:50051 -v /path/to/your/config.toml:/etc/astarte-message-hub/config.toml astarte-message-hub:latest
```

Comment:
What do you think to add the possibility to use also the environment variables?
```
docker run -e "DEVICE_ID="  -e "PAIRING_URL=<your pairing url>" -e "REALM=<realm name>"
```
https://github.com/astarte-platform/stream-qt5-test/blob/bbfc5b764402d90e79e90d8c364913717034473e/README.md?plain=1#L70
This allows to use one-line command without use to use/edit a file.
Perhaps this will involve something to change in message-hub code.

Reply:
It would be great. The implementation in the message-hub would be straight forward since `clap` has an `env` feature. I would add all the configuration flags as command line arguments and derive the `#[clap(env)]` on them.

I would have all the options be prefixed with `MSGHUB_<OPTION>` to prevent conflicts. What do you think?